client/src/index.js

```diff
@@ -6,7 +6,7 @@ import thunk from "redux-thunk";
 import "antd/dist/antd.css";
 import "antd-mobile/dist/antd-mobile.css";
 import "typeface-poppins";
-// import "typeface-work-sans";
+import "typeface-work-sans";
```

Comment:
@anniegiang eh, why would it or wouldn't it? I'm actually going to look at the fonts right now, I remember several people added fonts in different ways, for now as long as we only have each font once I guess it's ok.
The other PR about create user profile also included fonts in another component, and it wasn't working. I also wonder if we really need 3 custom fonts since it might have a big impact on performance.

Reply:
The ticket for performance improvements mentions we should use Google's CDN instead of importing libraries, but for now we could leave this in here and optimize it later